ui/src/pages/Lore/Lore.tsx

```diff
@@ -1,5 +1,7 @@
 export function Lore() {
   return (
-    <h2>THIS IS LORE PAGE</h2>
+    <div className="lore">
+      <h2>THIS IS LORE PAGE</h2>
```

Comment:
For this, create a bunch of text. Whether it be the anya text we use all over or somewhere else. I want to see what this looks like when we have a bunch of text